specutils/tests/test_analysis.py

```diff
@@ -467,6 +467,7 @@ def test_centroid(simulated_spectra):
 
     assert isinstance(spec_centroid, u.Quantity)
     assert np.allclose(spec_centroid.value, spec_centroid_expected.value)
+    assert hasattr(spec_centroid, 'uncertainty')
 
```

Comment:
```suggestion
quantity_allclose(spec_centroid.uncertainty, 4.4585e-06*u.um, rtol = 5e-5)
```

Reply:
Hold on, that actual fails...what did I mess up... 😆

Reply:
Maybe add a comment that the actual value has not been properly vetted yet (the math hasn't been reviewed, afaik).  But putting something in to ensure it doesn't accidentally change is probably still a good idea.